### ISSUE TITLE
a problem with importing these modules

### ISSUE THREAD

**Issue by yanzuotao** [AFeng-x/SMT#12]
Hello, author, how to solve the following problem:
![980ad3a8a6c2cf7adc6e0b2b919c0cd](https://github.com/AFeng-x/SMT/assets/72297553/27d99961-b9a0-43ad-b3cd-82e48ce32205)

**Comment by AFeng-x** [comment]
hi! You can delete these unrelated codes, for example:
`
from .build import build_loader as _build_loader
def build_loader(config, simmim=False, is_pretrain=False):
    if not simmim:
        return _build_loader(config)
`

**Comment by AFeng-x** [comment]
hi! You can delete these unrelated codes, for example:
`
from .build import build_loader as _build_loader 
def build_loader(config, simmim=False, is_pretrain=False):
       if not simmim: 
             return _build_loader(config)
`

**Comment by AFeng-x** [comment]
hi! You can delete these unrelated codes, for example:
from .build import build_loader as _build_loader 
def build_loader(config, simmim=False, is_pretrain=False):
            if not simmim: 
                   return _build_loader(config)

**Comment by yanzuotao** [comment]
Thank you for your reply. I have solved this problem according to your method

**Comment by yanzuotao** [comment]
Sorry to bother you again, can we apply the pre-trained model trained from scratch to the test set images and output the predicted images and metrics in this project?
